repo/packages/K/kubernetes/1/config.json

```diff
@@ -0,0 +1,195 @@
+          "type": "string",
+          "pattern": "^(?:\\.[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)+$"
+        },
+        "enable-dns": {
+          "default": "1",
+          "description": "Set to 1 to enable Kubernetes local DNS services, allows pods to locate services by name.",
+          "type": "string",
+          "pattern": "^1?$"
+        },
+        "framework-name": {
+          "default": "kubernetes",
+          "description": "Human-readable name to use when registering with the Mesos master.",
+          "type": "string",
+          "pattern": "^(?:[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)$"
+        },
+        "kube-dns-domain": {
+          "default": "kubernetes.local",
+          "description": "Domain used by pods for Kubernetes service lookup.",
+          "type": "string",
+          "pattern": "^(?:[a-z][a-z0-9]*?(?:-[a-z0-9]+)*?\\.)?(?:[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)$"
+        },
+        "kube-dns-service-ip": {
+          "default": "10.10.10.10",
+          "description": "Kubernetes DNS service portal IP address, should fall in service-cluster-ip-range network range.",
+          "type": "string",
+          "pattern": "^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$"
+        },
+        "kube-dns-replicas": {
+          "default": 1,
+          "description": "The number of kube-dns pods that Kubernetes will spin up at deploy time.",
+          "minimum": 1,
+          "type": "integer"
+        },
+        "kube-dns-nameservers": {
+          "description": "Comma-delimited list of host:ip entries that point to fallback nameservers. Only used if nameservers cannot be extracted from the slave's /etc/resolv.conf",
+          "type": "string",
+          "pattern": "^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?):(6553[0-5]|655[0-2][0-9]\\d|65[0-4](\\d){2}|6[0-4](\\d){3}|[1-5](\\d){4}|[1-9](\\d){0,3})(?:,(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?):(6553[0-5]|655[0-2][0-9]\\d|65[0-4](\\d){2}|6[0-4](\\d){3}|[1-5](\\d){4}|[1-9](\\d){0,3}))*$"
+        },
+        "service-cluster-ip-range": {
+          "default": "10.10.10.0/24",
+          "description": "Kubernetes DNS service portal IP address, should fall in service-cluster-ip-range network range.",
+          "type": "string",
+          "pattern": "^(?:(?:[0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}(?:[0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(?:\\/(?:[0-9]|[1-2][0-9]|3[0-2]))$"
+        },
+        "scheduler-service-ip": {
+          "default": "10.10.10.9",
+          "description": "Kubernetes-Mesos scheduler service portal IP address, should fall in service-cluster-ip-range network range.",
+          "type": "string",
+          "pattern": "^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$"
+        },
+        "disable-etcd-server": {
```

Comment:
if `enable-dns` becomes a bool then this should as well, for consistency. will require bootstrap.sh updates.